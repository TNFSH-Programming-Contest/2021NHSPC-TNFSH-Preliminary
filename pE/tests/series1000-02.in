999 998
3 4 21
4 5 15
5 6 18
7 8 14
9 10 20
10 11 21
12 7 24
13 14 27
14 15 21
16 3 14
8 17 6
6 18 28
15 12 30
11 13 15
18 9 28
17 19 7
20 16 13
21 20 6
22 21 9
23 24 17
25 22 5
24 25 16
26 23 7
27 26 21
28 27 19
29 28 6
30 29 10
19 31 18
32 33 29
34 35 20
36 30 18
37 36 12
31 32 16
38 39 9
33 40 21
41 42 6
40 43 6
43 44 12
42 45 26
46 47 25
44 48 14
49 50 5
45 51 23
35 52 25
53 54 7
48 55 23
54 56 18
57 49 11
58 59 16
60 61 21
59 62 10
47 41 28
63 64 16
52 63 25
65 66 8
67 68 19
62 69 11
64 70 20
39 34 24
71 57 15
72 73 17
69 67 14
55 74 9
51 65 24
75 71 7
61 76 28
74 72 30
56 77 11
50 78 20
78 60 10
79 80 17
73 46 15
76 58 17
77 75 23
81 82 25
80 37 25
70 53 25
83 38 24
84 83 8
85 79 23
66 81 13
82 86 11
68 85 18
87 88 10
88 84 15
86 89 8
90 91 26
92 93 29
94 95 22
96 97 19
89 94 12
98 96 27
95 99 21
100 101 10
91 100 18
101 102 17
103 98 25
99 104 10
93 105 10
105 103 13
104 106 8
107 108 16
97 109 6
110 90 19
111 110 19
106 112 10
113 114 12
112 115 13
116 117 5
109 116 24
114 92 28
118 119 29
115 120 12
121 113 23
122 123 6
124 107 27
108 125 10
102 122 11
123 126 18
127 121 9
128 127 11
120 111 18
119 87 11
126 129 13
125 130 25
131 132 18
132 133 30
129 131 27
133 134 28
135 118 15
130 136 20
137 138 19
117 124 15
139 137 23
140 128 29
138 135 17
136 141 12
142 140 13
134 143 20
143 144 6
145 146 25
146 139 11
141 147 18
147 145 27
148 142 26
149 150 27
151 152 28
153 154 13
152 153 16
154 155 15
156 157 29
157 149 27
155 158 12
144 159 7
160 161 11
162 163 12
164 165 17
165 166 11
167 156 25
168 167 27
166 162 28
163 169 24
170 160 15
171 170 11
150 164 8
172 173 15
159 174 23
175 176 9
177 178 18
176 177 15
179 148 9
180 181 5
173 182 5
174 151 19
183 184 17
185 180 5
186 172 12
182 187 28
161 179 7
187 171 23
178 183 9
188 186 30
158 175 23
189 185 18
181 168 23
169 190 19
184 191 25
192 193 11
190 194 9
191 195 28
196 188 6
197 198 22
199 200 11
201 202 20
198 199 24
203 204 29
204 205 9
206 207 9
208 209 28
210 211 18
212 213 8
214 192 22
215 216 7
193 217 9
202 196 29
218 219 24
195 220 25
219 221 14
222 223 24
216 212 6
211 197 22
209 224 14
225 189 13
224 226 21
217 227 27
228 203 24
229 222 10
230 225 19
231 232 9
200 233 21
205 234 5
235 215 24
236 206 18
232 237 30
223 238 15
239 235 11
240 241 15
242 243 14
194 244 23
244 245 21
246 247 19
248 249 29
250 248 6
251 252 6
252 253 23
207 254 28
255 256 18
256 257 16
258 239 9
213 259 21
260 231 22
261 214 30
221 262 23
226 263 24
253 242 28
264 240 22
265 230 28
237 266 22
267 268 24
269 201 24
249 270 24
257 271 28
262 267 10
272 273 27
274 275 16
247 272 12
276 208 12
241 277 28
277 278 17
271 265 6
279 280 14
270 261 21
233 281 22
263 228 19
280 282 16
283 284 30
285 274 23
254 251 18
220 255 9
286 287 6
288 246 14
289 290 13
291 292 16
259 285 29
293 264 20
281 294 6
234 295 16
227 296 21
294 229 26
287 283 5
273 236 18
278 297 10
282 258 10
298 299 10
297 291 15
275 218 29
295 293 7
296 300 8
301 286 16
238 302 27
303 276 16
284 298 27
268 288 9
304 305 21
306 307 20
308 309 13
299 303 23
243 310 6
311 304 13
312 313 17
302 314 29
315 316 17
316 306 19
317 318 8
319 320 30
321 322 8
323 324 16
325 326 25
292 311 5
327 325 27
328 289 22
300 319 13
307 210 26
329 330 9
314 331 18
332 333 6
305 334 19
335 269 25
322 279 22
336 337 16
338 339 28
337 340 6
341 342 19
309 260 9
343 336 19
245 315 26
313 344 24
345 327 20
334 328 12
346 250 30
347 348 28
290 349 9
344 343 18
350 351 12
333 352 27
340 353 24
310 308 13
326 350 16
349 354 13
339 355 20
352 356 10
353 332 9
331 357 21
355 358 29
320 317 29
356 359 19
360 361 21
357 362 21
361 363 9
364 346 5
362 347 11
365 366 12
367 301 5
354 338 10
359 368 5
348 369 19
370 345 6
358 365 19
369 371 14
372 364 26
324 373 9
371 374 9
375 376 28
377 378 5
318 341 11
379 380 7
381 382 12
383 384 27
266 372 23
376 385 5
386 321 7
351 387 6
388 389 30
382 390 24
391 370 11
380 386 15
385 312 6
330 392 22
393 329 13
394 391 11
389 379 16
395 381 24
363 367 22
396 375 15
397 398 30
392 399 20
400 383 21
401 335 17
402 400 7
373 377 19
387 402 7
398 403 28
390 323 23
366 397 25
384 404 13
378 405 5
404 406 19
406 407 20
368 408 5
409 395 10
342 394 25
410 409 10
411 410 19
405 360 18
412 413 14
407 396 21
374 411 12
414 401 29
403 415 24
416 393 14
408 416 20
413 417 5
418 419 9
420 388 22
399 412 30
419 421 27
417 422 26
423 424 22
424 418 14
425 414 22
426 423 17
415 427 23
428 426 17
421 429 5
427 428 20
430 431 12
432 420 18
431 432 11
433 434 12
422 433 26
434 435 18
436 425 24
437 430 9
435 436 5
429 438 12
438 439 6
440 437 26
441 440 13
442 443 25
439 444 17
445 446 21
447 448 17
449 441 21
450 451 8
452 447 9
453 452 24
454 455 5
446 456 7
457 458 30
459 460 28
461 449 10
462 463 17
464 465 25
466 450 25
460 467 11
468 469 11
470 459 23
443 471 18
472 464 25
473 474 11
471 468 29
475 472 25
448 475 15
463 461 28
458 476 8
477 454 25
478 479 17
469 462 25
480 481 28
482 445 9
483 484 11
467 478 18
479 485 13
486 487 17
481 488 11
488 489 12
487 490 18
491 492 25
490 473 13
493 494 28
495 496 27
497 482 17
444 498 8
474 480 24
499 500 18
498 501 7
492 502 12
489 493 16
500 503 17
451 504 25
504 499 23
503 505 25
506 507 5
484 457 22
501 453 6
508 495 29
507 483 6
509 510 19
511 497 28
512 442 28
456 470 22
476 513 9
510 514 26
494 515 20
496 506 15
516 509 24
517 518 9
519 491 20
455 508 6
520 521 18
522 477 10
523 516 27
514 524 5
525 526 30
527 519 29
505 528 6
528 527 7
502 522 8
465 529 23
526 466 7
521 523 16
518 530 18
530 520 7
513 531 27
485 532 7
515 533 14
531 517 6
534 511 15
535 534 9
533 536 15
537 512 26
529 486 6
532 537 27
538 535 16
524 539 11
540 541 22
541 525 19
539 538 10
542 543 29
544 542 9
545 544 19
546 547 29
548 549 16
549 550 8
551 540 26
552 546 16
553 551 14
536 552 9
547 554 29
555 556 26
550 555 5
556 557 24
557 558 26
559 553 8
560 561 27
561 545 23
562 563 6
563 564 20
554 565 22
564 559 24
566 567 14
568 560 27
569 562 9
570 548 9
567 568 28
571 569 17
558 571 27
572 570 20
543 573 24
565 574 13
575 572 14
576 577 6
578 566 7
577 579 20
573 575 14
580 581 21
579 578 8
582 583 17
574 582 28
581 584 24
585 586 18
586 587 17
584 585 14
588 580 13
589 588 24
590 591 17
587 590 26
592 593 23
591 594 20
595 596 27
596 597 18
583 598 22
593 595 8
599 600 12
597 589 14
598 592 13
601 576 24
602 601 12
603 599 15
604 605 30
606 607 18
607 608 20
609 610 10
611 612 25
613 606 22
614 611 25
594 615 24
615 616 21
608 617 7
610 602 16
616 618 28
617 609 7
600 619 11
612 603 26
620 613 16
619 604 30
621 620 12
618 622 15
623 624 13
605 625 16
622 614 22
624 626 18
625 627 16
628 621 26
629 630 19
630 631 29
626 632 9
631 628 26
627 623 7
632 633 26
633 634 26
635 629 29
636 635 23
637 636 14
638 639 12
639 637 6
634 640 11
641 638 18
642 643 7
644 645 15
646 647 10
648 649 27
643 650 21
651 652 18
650 653 20
645 651 14
654 655 24
653 641 7
656 657 30
658 646 6
659 648 30
657 660 19
640 659 17
661 662 10
663 664 5
652 665 19
662 663 8
666 667 18
668 669 5
665 670 11
655 671 28
672 668 28
673 672 8
674 675 28
676 677 21
678 642 18
679 674 5
671 680 25
667 681 11
669 682 14
649 683 14
684 685 10
686 687 22
682 678 12
688 689 26
680 666 28
690 691 20
683 692 29
693 676 9
670 694 8
695 690 23
696 673 22
697 656 26
698 695 19
681 644 10
660 661 17
687 699 18
689 700 20
691 697 18
675 654 22
699 696 27
701 684 11
702 703 18
704 658 30
705 706 24
694 707 20
708 679 13
709 710 11
710 698 26
700 711 12
712 713 14
714 715 30
713 688 9
664 716 24
716 717 20
703 704 29
707 712 5
718 702 8
685 709 14
719 714 30
647 720 13
715 708 23
692 721 8
677 686 20
706 722 29
717 718 26
723 724 24
725 719 15
726 727 16
728 725 5
729 723 13
730 705 21
720 693 22
731 701 27
724 732 24
727 728 7
732 733 15
734 735 24
711 736 19
737 730 21
722 738 19
739 740 30
741 742 9
743 744 27
740 745 11
745 746 9
742 726 26
747 729 10
744 747 24
748 731 15
749 734 20
735 750 6
751 748 20
752 749 10
738 753 25
750 751 20
754 752 11
755 741 9
746 756 18
733 757 17
753 755 12
736 758 24
756 759 10
760 754 27
761 762 9
763 743 23
721 761 8
758 764 24
762 765 14
765 739 10
766 767 29
759 768 30
757 737 10
768 766 26
769 763 25
770 760 6
771 770 7
767 769 18
772 773 14
773 771 11
774 775 28
776 777 27
775 772 5
777 774 9
778 779 26
764 780 30
781 782 12
783 776 19
784 781 29
780 785 27
786 787 30
788 784 10
789 788 26
785 790 5
787 778 20
791 792 7
793 783 22
782 791 12
779 789 13
794 786 10
795 794 24
792 796 28
797 793 23
798 795 23
799 797 11
800 801 22
796 802 26
803 799 10
790 798 10
802 800 17
804 805 8
806 807 26
801 808 12
805 803 5
807 809 26
808 810 25
811 812 22
809 813 8
814 815 11
816 817 21
818 816 18
819 820 26
821 819 15
812 814 29
822 823 21
815 824 18
813 822 14
810 821 6
823 825 17
817 804 8
826 806 8
824 827 21
820 826 8
827 828 16
828 829 7
825 830 8
831 811 10
830 831 12
832 833 6
834 832 26
835 836 30
836 837 7
838 839 11
833 838 25
837 840 26
829 841 26
841 834 13
842 843 27
839 835 12
843 844 6
844 818 22
845 842 25
840 846 20
847 848 23
849 847 12
850 849 14
848 845 24
851 850 28
852 851 20
846 853 19
854 852 29
853 855 19
856 857 23
857 854 18
858 856 12
859 860 8
860 858 30
861 859 8
855 862 5
863 861 26
862 864 30
865 866 6
867 863 27
868 869 16
870 867 5
871 872 24
866 873 17
864 874 29
875 870 26
874 876 19
869 877 25
876 878 26
873 871 8
877 865 14
879 875 15
878 880 10
880 868 21
881 879 13
882 883 27
884 881 28
883 884 22
872 885 20
886 887 24
888 886 22
889 882 28
890 891 23
892 893 24
894 895 19
896 897 8
895 898 9
897 894 8
899 890 22
898 900 24
893 899 19
887 889 13
901 892 28
885 896 17
902 903 28
904 905 13
906 907 12
905 906 9
891 908 6
909 910 9
911 902 5
907 888 13
910 901 27
903 912 6
908 913 7
914 915 12
916 909 30
917 914 7
918 919 14
920 918 6
915 921 15
922 923 18
919 924 10
925 904 29
926 911 28
913 927 18
928 922 30
923 929 15
927 930 13
929 926 12
931 932 15
912 917 15
930 931 30
924 925 10
933 916 23
900 928 15
934 933 10
932 935 18
921 936 20
936 937 19
935 920 25
938 934 14
937 939 9
939 940 19
941 942 9
942 943 11
944 945 7
945 946 21
943 947 28
947 938 24
940 948 17
946 941 22
949 950 13
951 944 13
948 949 23
950 952 18
952 953 25
954 955 25
953 956 21
956 954 13
957 958 19
959 951 11
960 959 9
961 962 24
963 964 25
964 957 6
962 960 27
965 963 12
958 966 26
955 967 23
968 969 12
970 961 7
971 972 23
967 973 18
974 975 10
972 976 26
977 978 25
978 970 20
979 974 21
980 981 19
982 971 24
983 984 18
985 986 11
986 980 5
984 987 25
988 979 29
989 990 25
987 982 23
976 989 27
991 968 28
992 985 15
973 993 14
990 988 24
969 977 17
966 983 24
993 994 12
995 991 15
994 992 14
996 997 26
998 965 8
981 998 12
975 996 26
999 2 7
997 999 7
1 995 29
