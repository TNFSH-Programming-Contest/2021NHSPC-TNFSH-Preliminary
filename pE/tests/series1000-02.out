17449.0000000000000000000
